.github/workflows/pull-request.yml

```
@@ -0,0 +1,66 @@
+name: PR Workflow
+on: [pull_request]
+
+jobs:
+  PR-Greeting:
+    name: Greeting Message to user
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/first-interaction@v1
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          pr-message: "Congratulations on making your first PR! :confetti_ball: If you haven't already, check out our [Contributing Guidelines](https://github.com/PalisadoesFoundation/talawa/blob/master/CONTRIBUTING.md) and [PR Reporting Guidelines](https://github.com/PalisadoesFoundation/talawa/blob/master/PR-guidelines.md) to ensure that you are following our guidelines for contributing and creating PR."
+  
+  Styler:
+    name: Checks Styling of code
```

Similarly, please adjust this to
```suggestion
    name: Checks Code Style
```

As the grammar here needs to be adjusted slightly

---

Done!